src/javascript/app/pages/user/view_popup/view_popup.js

```diff
@@ -135,7 +135,7 @@ const ViewPopup = (() => {
     const update = () => {
         const final_price       = contract.sell_price || contract.bid_price;
         const is_started        = !contract.is_forward_starting || contract.current_spot_time > contract.date_start;
-        const user_sold         = contract.sell_time && contract.sell_time < contract.date_expiry;
+        const user_sold         = contract.sell_spot_time && +contract.sell_spot_time < contract.date_expiry;
```

Comment:
I'm worried this will break for other contracts, because it's being used everywhere. @binary-hamedanchi ?

Reply:
Since this is strictly about _sold by user_, we should check `sell_time` as the time which sell command triggered.
`sell_spot_time` is the time of the spot that contract is actually sold, which can happen after expiry.
If `sell_time` is after `date_expiry` then we can assume that it's not sold _by user_.